### DISCUSSION TITLE
Is it possible to type in precise pixel values for the width and height?

### DISCUSSION THREAD

**Discussion by flowerdealer** [hlky/stable-diffusion#181] [Q&A]
Can't figure out how to get exact image aspect ratios.

**Comment by JoshuaKimsey** [comment]
Unfortunately, I don't think this is possible. The pixel counts have to be divisible by a certain number (I forget what it is, I think it's 64), meaning that only so many options are possible for the sizing. If you are going for a certain aspect ratio, it would probably be best to go slightly larger than desired and then crop the image to the desired ratio.

**Reply by flowerdealer** [reply]
Ok, it's just a bit weird as the new UI seems to have an apsect ratio measurement which shows some common rations like 16:9, but you can't actually match them with the pixel sliders, or not that I can see, at least at lower resolutions.

**Reply by JoshuaKimsey** [reply]
Well, I think that exists so it gives you an idea of what you're going for, even if Stable Diffusion can't achieve a perfect 4:3 or 16:9 ratio.
